TRACONBoundaries.geojson

```diff
@@ -480,8 +480,19 @@
+    {"type":"Feature","properties":{"id":"UWKE","prefix":["UWKE_R"],"name":"Begishevo Radar"},"geometry":{"type":"MultiPolygon","coordinates":[[[[52.09249999999997,55.924451328639655],[52.19202178223816,55.91998173336195],[52.28909243411829,55.90668605060536],[52.38132130534177,55.884900494791566],[52.46643718923019,55.855175216417955],[52.542344296580836,55.81825936726741],[52.60717385083274,55.77508091994327],[52.65933003631528,55.726721919184705],[52.697529182847234,55.67438996922052],[52.72083124574351,55.61938684735831],[52.72866283403323,55.56307517852576],[52.72083124574351,55.50684411118263],[52.697529182847234,55.45207490695461],[52.65933003631528,55.40010730113925],[52.60717385083274,55.352207416214085],[52.542344296580836,55.30953792281132],[52.46643718923019,55.273131048718085],[52.38132130534177,55.2438649415125],[52.28909243411829,55.222443798244896],[52.19202178223816,55.2093820884571],[52.09249999999997,55.204993115804776],[51.99297821776179,55.2093820884571],[51.89590756588177,55.222443798244896],[51.80367869465829,55.2438649415125],[51.718562810769754,55.273131048718085],[51.64265570341911,55.30953792281132],[51.57782614916721,55.352207416214085],[51.52566996368466,55.40010730113925],[51.48747081715271,55.45207490695461],[51.464168754256434,55.50684411118263],[51.456337165966715,55.56307517852576],[51.464168754256434,55.61938684735831],[51.48747081715271,55.67438996922052],[51.52566996368466,55.726721919184705],[51.57782614916721,55.77508091994327],[51.64265570341911,55.81825936726741],[51.718562810769754,55.855175216417955],[51.80367869465829,55.884900494791566],[51.89590756588177,55.90668605060536],[51.99297821776179,55.91998173336195],[52.09249999999997,55.924451328639655]]]]}},
+    {"type":"Feature","properties":{"id":"UWLL","prefix":["UWLL"],"name":"Ulyanovsk Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[48.25,54.86666666666667],[48.775,54.82944444444445],[49.54138888888889,54.77111111111111],[49.64277777777778,54.34277777777778],[49.17833333333333,54.04111111111111],[48.865,53.766666666666666],[47.84444444444445,53.70138888888889],[47.20055555555556,53.78],[46.858333333333334,54.24666666666667],[46.583333333333336,54.81666666666667],[48.25,54.86666666666667]]]]}},
```

Comment:
Looks for the most part very good, there is just one minor polygon I suspect is not correct. Reference the image below, but to me, this does not make sense airspace wise to have this weird outlier that goes to the corner. It would make more sense for it to end in the middle of the line further south-west, no? 
![Screen Shot 2022-09-22 at 12 51 40](https://user-images.githubusercontent.com/967239/191728435-5d0f3bac-8d84-44f0-bb3a-2ff22f192e8b.png)

Reply:
That coordinates are verified with AIP of RU. Will double check that anyway.